doc/nrf/ug_dev_model.rst

```diff
@@ -357,15 +357,68 @@ This workflow is particularly beneficial if your application is split among mult
 
 In order to implement this approach you first need to create a manifest repository of your own, which just means a repository that contains a :file:`west.yml` manifest file in its root.
 Next you must populate the manifest file with the list of repositories and their revisions.
-In general, the easiest thing to do is to take the :file:`west.yml` in `fw-nrfconnect-nrf`_ and copy its entries directly as a starting point.
+
+In general, the easiest thing to do is to import the :file:`west.yml` in `fw-nrfconnect-nrf`_ using west's manifest imports feature, like this:
+
+.. code-block:: yaml
+
+   # Example application-specific west.yml, using manifest imports.
+   manifest:
+     remotes:
+       - name: ncs
+         url-base: https://github.com/NordicPlayground
+     projects:
+       - name: nrf
+         remote: ncs
+         revision: v1.2.0
+         import: true
+     self:
+       path: application
+
 Then, make the following changes:
 
-  * Add an entry for `fw-nrfconnect-nrf`_ or a forked version of it, if applicable.
-  * Point the entries of any |NCS| repositories that you have forked to your fork and fork revision.
-  * Add any entries for repositories that you need and that are not part of the |NCS|.
+  * Point the entries of any |NCS| repositories that you have forked to your fork and fork revision, by adding them to the ``projects`` list using a new remote.
+  * Add any entries for repositories that you need that are not part of the |NCS|.
+
+For example:
 
-Once you have your new manifest repository, you can use it with west just like you would use `fw-nrfconnect-nrf`_ when :ref:`getting <dm-wf-get-ncs>` and later :ref:`updating <dm-wf-update-ncs>` the source code.
-You just need to replace ``fw-nrfconnect-nrf`` and ``nrf`` with whatever repository name and path you have chosen for your manifest repository.
+.. code-block:: yaml
+
+   # Example your-application/west.yml, using manifest imports, with
+   # an NCS fork and a separate module
+   manifest:
+     remotes:
+       - name: ncs
+         url-base: https://github.com/NordicPlayground
+       - name: your-remote
+         url-base: https://github.com/your-name
+     projects:
+       - name: nrf
+         remote: ncs
+         revision: v1.2.0
+         import: true
+       - name: nrfx
+         remote: your-remote
+         revision: your-nrfx-fork-SHA
+       - name: your-custom-library
+         remote: your-remote
+         revision: your-library-SHA
```

Comment:
just a small thought here.
Some downstream users of NCS might prefer to track a branch here to avoid constantly updating the SHA for internal development, and this is fully allowed.

So as a minimum we should inform them of this possibility, either directly or referring to `west` documentation.

Reply:
+1 on this, I completely agree

Reply:
will change to:

```
         revision: your-library-SHA-or-branch
```